courses/guidance/toc.md

```diff
@@ -0,0 +1,49 @@
+Outline of the course.
+
+1. Overview and Objectives
```

Comment:
I would add the objectives in here to make sure they match the outline. If they don't, we need to change the content or change the objectives.

Identify and locate security guidance for a software component.
    Differentiate between different types of security guidance, such as SRGs and STIGs.
    Create tailored security guidance using Vulcan.
    Classify security controls as Applicable - Configurable, Applicable - Inherently Meets, Applicable - Does Not Meet, Not Applicable, or Not Yet Determined for a given software component.
    Export security guidance as InSpec stubs to assist in automated security validation.

[#](https://mitre.github.io/saf-training/courses/guidance/#_1-2-the-road-to-security-automation)

Reply:
Also, you may want to update the objectives if they don't cover everything you have

Reply:
Fair -- added more objectives re: the DISA STIG process and how all this fits into the ATO process